tests/tests_cache/writing/TextTest/Text/space.svg

```diff
@@ -6,7 +6,7 @@
 <path style="stroke:none;" d="M 1.25 0 L 1.25 -6.25 L 6.25 -6.25 L 6.25 0 Z M 1.40625 -0.15625 L 6.09375 -0.15625 L 6.09375 -6.09375 L 1.40625 -6.09375 Z M 1.40625 -0.15625 "/>
 </symbol>
 <symbol overflow="visible" id="glyph0-1">
-<path style="stroke:none;" d="M -0.152344 1.988281 L -0.152344 1.351562 L 5.671875 1.351562 L 5.671875 1.988281 Z M -0.152344 1.988281 "/>
+<path style="stroke:none;" d="M -0.152344 1.988281 L -0.152344 1.351563 L 5.671875 1.351563 L 5.671875 1.988281 Z M -0.152344 1.988281 "/>
```

Comment:
Comment: whenever you run the tests locally, this file is always changed. We don't know why and we don't know how to change it yet. See #230.

@huguesdevimeux is merging this change to space.svg safe? (BTW, until we fix #230, we should add space.svg to .gitignore maybe?)

Reply:
Albeit unwanted this change seems to be fine. I don't really see how it could cause trouble as it is re-generated every time.